[No QA] Create Expensify-Card-Perks.md (#29407)
Expensify/App: docs/articles/expensify-classic/expensify-card/Expensify-Card-Perks.md

---

# Overview
The Expensify Card is packed with perks, both native to our Card program and through exclusive discounts with partnering solutions. The Expensify Card’s primary perks include: 
- Access to our premiere Expensify Lounge (with more locations coming soon)
- Swipe to Win, where every swipe has a chance to win fun personalized gifts for you and your closest friends and family members
- And unbeatable cash back incentive with each swipe
Below, we’ll cover all of our exclusive offers in more detail and how to claim discounts with our partners.

# Expensify Card Perks

## Access to the Expensify Lounge
Our [world-class lounge](https://use.expensify.com/lounge) is now open for Expensify members and guests to enjoy! 

We invite you to visit our sleek San Francisco lounge, where sweeping city views provide the perfect backdrop for a morning coffee to start your day. 

Enjoy complimentary cocktails and snacks in a vibrant atmosphere with blazing-fast WiFi. Whether you want a place to focus on work, socialize with other members, or simply kick back and relax – our lounge is ready and waiting to welcome you.

> You can sign up for free [here](https://community.expensify.com/home/leaving?allowTrusted=1&target=https%3A%2F%2Fuse.expensify.com%2F) if you’re not an Expensify member. If you have any questions, reach out to concierge@expensify.com and [check](https://community.expensify.com/home/leaving?allowTrusted=1&target=https%3A%2F%2Fuse.expensify.com%2Flounges) this out for more info.

Reviewer: This link goes through community.expensify.com just to re-route to use.expensify.com. Since we're trying to deprecate community, let's update this link accordingly:

```suggestion
You can sign up for free [here](https://use.expensify.com) if you’re not an Expensify member. If you have any questions, reach out to concierge@expensify.com and [check this out](https://use.expensify.com/lounge) for more info.
```

## Swipe to Win

> Swipe to Win is a new [Expensify Card](https://community.expensify.com/home/leaving?allowTrusted=1&target=https%3A%2F%2Fuse.expensify.com%2Fcard) perk that gives cardholders the chance to send a gift to a friend, family member, or essential worker on the frontlines!

Reviewer: ```suggestion
Swipe to Win is a new [Expensify Card](https://use.expensify.com/company-credit-card) perk that gives cardholders the chance to send a gift to a friend, family member, or essential worker on the frontlines!
```

> Winners can choose to Send a Smile or Send a Laugh. To start, we’re offering one gift per option:

Reviewer: ```suggestion
Winners can choose to _Send a Smile_ or _Send a Laugh_. To start, we’re offering one gift per option:
```

- **Send A Smile:** Champagne by Expensify

> - **Send a Laugh:** Jenge Set

Reviewer: ```suggestion
- **Send a Laugh:** Jenga Set
```

**How to Participate**
It’s easy! Once you have an Expensify Card, you just need to start using it. With each swipe, you're automatically entered to win and have a 1 in 250 chance of getting a prize!

**How will I know if I’ve won?**
Winners will be notified immediately via the Expensify app, and receive additional instructions on how to choose and send their desired gift. 

If you don't have Expensify notifications turned on yet, here are some helpful guides:

> - [Apple Notification Preferences](https://community.expensify.com/home/leaving?allowTrusted=1&target=https%3A%2F%2Fsupport.apple.com%2Fen-us%2FHT201925)

Reviewer: ```suggestion
- [Apple Notification Preferences](https://support.apple.com/en-us/HT201925)
```

> - [Android Notification Preferences](https://community.expensify.com/home/leaving?allowTrusted=1&target=https%3A%2F%2Fsupport.google.com%2Fandroid%2Fanswer%2F9079661%3Fhl%3Den)

Reviewer: These links take you to a community page like this:

<img width="1512" alt="image" src="https://github.com/Expensify/App/assets/47436092/33ab5fb6-02ec-4b22-9a24-c01b19f489e2">

Let's just jump straight to the destination:

https://support.google.com/android/answer/9079661?hl=en

# Partner Specific Perks

## Amazon AWS
Whether you are a two-person startup launching a new company or a venture-backed startup, we all could use a little relief in these difficult times.  AWS Activate provides you with access to the resources you need to quickly get started on AWS - including free credits, technical support, and training. 

All Expensify customers that have adopted The Expensify Card qualify when they add their Expensify Card for billing with AWS!

> **Apply now by going [to this link](https://community.expensify.com/home/leaving?allowTrusted=1&target=https%3A%2F%2Faws.amazon.com%2Factivate%2Fportfolio-signup) and using the OrgID: 0qyIA (Case Sensitive)**

Reviewer: ```suggestion
**Apply now by going [to this link](https://aws.amazon.com/startups/credits) and using the OrgID: 0qyIA (Case Sensitive)**
```

> The full details on the AWS Activate program can be found in AWS's [terms & conditions](https://community.expensify.com/home/leaving?allowTrusted=1&target=https%3A%2F%2Faws.amazon.com%2Factivate%2Fterms%2F) and the [Activate FAQs](https://community.expensify.com/home/leaving?allowTrusted=1&target=https%3A%2F%2Faws.amazon.com%2Factivate%2Ffaq%2F).

Reviewer: ```suggestion
The full details on the AWS Activate program can be found in AWS's [terms & conditions](https://aws.amazon.com/activate/terms/) and the [Activate FAQs](https://aws.amazon.com/startups/faq).
```

## Lamar Advertising
Lamar provides out-of-home advertising space for clients on billboards, digital, airport displays, transit, and highway logo signs.

**Receive at minimum a 10% discount on your first campaign.**

**How do redeem:** Contact Expensify’s dedicated account manager, Lisa Kane, and mention you’re an Expensify cardholder.

Email: lkane@lamar.com

## Carta
Simplify equity management with Carta.

**Receive a 20% first-year discount and waived implementation fees for Carta.**

**How to redeem:** Sign up using your Expensify Card

## Pilot
Pilot specializes in bookkeeping and tax prep for startups and e-commerce providers. When you work with Pilot, you’re paired with a dedicated finance expert who takes the work off your plate and is on hand to answer your questions.

**20% off the first 6-months of Pilot Core**

**How to redeem:** Sign-up using your Expensify Card.

## Spotlight Reporting
The integrated cloud reporting and forecasting tool that allows you to create insights for better business decisions. Designed by Accountants, for Accountants

**20% discount off your subscription for the first 6 months, plus one free seat to Spotlight Certification.**

> **How do redeem:** Sign up using your Expensify Card.

Reviewer: ```suggestion
**How to redeem:** Sign up using your Expensify Card.
```

## Guideline
Guideline's full-service 401(k) plans make it easier and more affordable to offer your employees the retirement benefits they deserve.

**Receive 3 months free.**

**How to redeem:** Sign up using your Expensify Card.

## Gusto
Gusto's people platform helps businesses like yours onboard, pay, insure, and support your hardworking team. Payroll, benefits, and more

> **3-months free service** 

Reviewer: ```suggestion
**3 months free service** 
```

**How to redeem with your Expensify Card:** Use the code at EXPENSIFY at the time of checkout.

## Stripe Atlas
Stripe Atlas helps removes obstacles typically associated with starting a business so you can build your startup from anywhere in the world. 

**Receive $100 off Stripe Atlas and get access to a startup toolkit and special offers on additional Strip Atlas services.**

**How to redeem:** Sign up with your Expensify Card.

# FAQ

## Where is the Expensify Lounge?
The Expensify Lounge is located on the 16th floor of 88 Kearny Street in San Francisco, California, 94108. This is currently our only lounge location, but keep an eye out for more work lounges popping up soon!

## When is the Expensify Lounge open?
The lounge is open 8 a.m. to 6 p.m. from Monday through Friday, except for national holidays. Capacity is limited, and we are admitting loungers on a first-come, first-served basis, so make sure to get there early!

## Who can use the lounge workplace?
Customers with an Expensify subscription can use Expensify’s lounge workplace, and any partner who has completed [ExpensifyApproved! University!](https://university.expensify.com/users/sign_in?next=%2Fdashboard)

> 

Reviewer: Need to delete everything after this line
